message/internal_msg_builder.go

```diff
@@ -473,11 +435,13 @@ func (m *CrossChainAppRequestFailed) GetRequestId() uint32 {
 	return m.RequestID
 }
 
-func InternalCrossChainAppRequestFailed(
+func InternalCrossChainAppError(
 	nodeID ids.NodeID,
 	sourceChainID ids.ID,
 	destinationChainID ids.ID,
 	requestID uint32,
+	errorCode int32,
+	errorMessage string,
 ) InboundMessage {
 	return &inboundMessage{
```

Comment:
Should we change `CrossChainAppRequestFailedOp` to `CrossChainAppErrorOp`?

Reply:
Yeah we can rename these ops